api/src/main/java/org/pmiops/workbench/api/ConceptSetsController.java

```diff
@@ -358,6 +360,63 @@ private void addConceptsToSet(
     return ResponseEntity.ok(toClientConceptSet(dbConceptSet));
   }
 
+  @Override
+  public ResponseEntity<ConceptSet> copyConceptSet(
+      String fromWorkspaceNamespace,
+      String fromWorkspaceId,
+      String fromConceptSetId,
+      CopyRequest copyRequest) {
```

Comment:
Could the `CopyRequest` object contain the origin namespace, workspace and concept set?